src/SDKs/NetApp/NetApp.Tests/Properties/AssemblyInfo.cs

```diff
@@ -35,5 +35,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
Test project versions are always set to `1.0.0.0` and never modified

Reply:
ok. Updated.